pkg/cmd/cmd.go

```diff
@@ -78,7 +79,7 @@ func NewJXCommand(f clients.Factory, in terminal.FileReader, out terminal.FileWr
 	configureViper()
 	rootCommand := &cobra.Command{
 		Use:              "jx",
-		Short:            "jx is a command line tool for working with Jenkins X",
+		Short:            i18n.T("jx is a command line tool for working with Jenkins X"),
```

Comment:
Would the plan be to internationalise just the help texts or also error messages? I am in particular concerned about error messages. We heavily rely on grepping for error messages when users report issues. This might get much harder in the case error messages are internationalised. What's your take on that?

Another question which comes to my mind - it seems you are suggesting to use the English text as message key, right. The benefit, of course, is that this was the default text is still directly in the code. On the other hand, the actual text is duplicated between the code and the message bundle where it serves as id. What happens if one fixes a typo in the code. That will make the translation fail for this message, right? Does the tooling (eg go-xgettext) handle this in some form? Can one easily detect when message IDs are out of sync?

Reply:
Thanks for your deep thoughts.

In my opinion, international for help text is the primary task. We can generate the help document base on the different languages. Error messages are complicated, I agree with you. But the internationalized error messages are also helpful and useful. We can request the users provide the 
 error message as English or other language bases on the main communication language.

About the second question, I think that we can figure a way to detect the changes in the message keys. But I don't the solution now. I mean it should be possible to detect them.